newIDE/app/src/EventsFunctionsExtensionEditor/index.js

```diff
@@ -1410,9 +1410,11 @@ export default class EventsFunctionsExtensionEditor extends React.Component<
                       )
                     }
                     initialNodes={
-                      isDev !==
+                      isDev ===
```

Comment:
I'm not sure to follow why isDev is compared to mosaicContainsNode? Any chance you can rework this ternary?

Reply:
In development mode, the object list must be shown: the mosaic must be reset to default if the save settings don't contain it.
This will be removed as soon as we release the events-based objects editor.